dds_cli/project_status.py

```diff
@@ -148,6 +148,112 @@ def update_status(self, new_status, deadline=None, is_aborted=False, no_mail=Fal
+        #       'Title': 'First Project'
+        #       },
+        #   'project_status': {
+        #       'current_deadline': 'Sat, 04 Nov 2023 23:59:59 GMT',
+        #       'current_status': 'Available'},
+        #   'warning': 'Operation must be confirmed before proceding.'
+        #   }
+
+        # Extract default unit days and current deadline
+        default_unit_days = response_json.get("default_unit_days")
+        current_deadline = response_json.get("project_status").get("current_deadline")
+
+        # print information about the project status and table with the project info
+        print_info = (
+            f"\nCurrent deadline: [b][green]{current_deadline}[/green][/b]\n"
+            f"Default deadline extension: [b][green]{default_unit_days}[/green][/b] days\n"
+        )
+        table = self.generate_project_table(project_info=response_json.get("project_info"))
+        dds_cli.utils.console.print(table)
+        dds_cli.utils.console.print(print_info)
+
+        # First question, number of days to extend the deadline
+        prompt_question = (
+            f"Enter the number of days you want to extend the project, "
+            f"the number of days has to be equal or same as "
+            f"[b][green]{default_unit_days}[/green][/b].\n"
+            f"Or leave it empty to apply the default "
+            f"[b][green]{default_unit_days} days [/green][/b]"
+        )
+
+        dds_cli.utils.console.print(prompt_question)
+        extend_deadline = rich.prompt.Prompt.ask("-")
+        if not extend_deadline:
+            # Set extend_deadline to default
+            extend_deadline = default_unit_days
+        try:
+            # the input was an string --> convert to integer
+            extend_deadline = int(extend_deadline)
+            if extend_deadline > default_unit_days:
+                dds_cli.utils.console.print(
+                    "\n[b][red]The number of days has to be lower than "
+                    "the default deadline extension number[/b][/red]\n"
+                )
+                LOG.info("Exiting the function, try again")
+                sys.exit(0)
+
+        except ValueError:
+            dds_cli.utils.console.print(
+                "\n[b][red]Remember to write the number of days "
+                "using numbers (dont use letters)[/b][/red]\n"
+            )
+            LOG.info("Exiting the function, try again")
+            sys.exit(0)
+
+        # Second question, confirm operation
+        prompt_question = (
+            f"\n\n[b][blue]Are you sure [/b][/blue]you want to perform this operation?. "
+            f"\nThis will extend the deadline by [b][blue]{extend_deadline} days[/b][/blue]."
+            "\nYou can only extend the data availability a maximum of "
+            "[b][blue]3 times[/b][/blue], this consumes one of those times."
+        )
+
+        dds_cli.utils.console.print(prompt_question)
+        if not rich.prompt.Confirm.ask("-"):
```

Comment:
* We don't need the print here either, just use the `ask` to print it and ask.
* Also print the new deadline (the date) 
```suggestion
        # Second question, confirm operation
        if not rich.prompt.Confirm.ask(
            f"\nThe new deadline for project '{<print project ID here>}' will be: <add `extend_deadline` to the currentl deadline>"
            f"\n\n[b][blue]Are you sure [/b][/blue]you want to perform this operation?. "
            "\nYou can only extend the data availability a maximum of "
            "[b][blue]3 times[/b][/blue], this consumes one of those times."
        ):
```